nservicebus/containers/injecting-ibus.md

```diff
@@ -6,6 +6,8 @@ redirects:
  - nservicebus/how-do-i-get-a-reference-to-ibus-in-my-message-handler
 ---
 
+NOTE: In Version 6, the `IBus` interface has been deprecated and removed. Use the `IMessageHandlerContext` interface instead. For more information, please consult the [upgrade guide](/nservicebus/upgrades/5to6.md).
```

Comment:
Should this note mention `IBusSession` since it's used to replace `IBus` as well?